templates/kube-oidc-proxy.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: kubeaddons.mesosphere.io/v1alpha1
+kind: Addon
+metadata:
+  name: kube-oidc-proxy
+  namespace: kubeaddons
+  labels:
+    kubeaddons.mesosphere.io/name: kube-oidc-proxy
+  annotations:
+    appversion.kubeaddons.mesosphere.io/kube-oidc-proxy: "v0.1.1"
+    values.chart.helm.kubeaddons.mesosphere.io/kube-oidc-proxy: "https://raw.githubusercontent.com/mesosphere/charts/fb01c5b81bb77e78d3a9ff5f5d8ec4b2703c5fb4/staging/kube-oidc-proxy/values.yaml"
+spec:
+  kubernetes:
+    minSupportedVersion: v1.15.0
+  cloudProvider:
+    - name: aws
+      enabled: true
+    - name: docker
+      enabled: true
+    - name: none
+      enabled: true
+  requires:
+    matchLabels:
+      kubeaddons.mesosphere.io/provides: ingresscontroller
+  chartReference:
+    chart: kube-oidc-proxy
```

Comment:
this chart cannot be upgraded multiple times due to the use of hook resources
https://github.com/mesosphere/charts/blob/dev/staging/kube-oidc-proxy/templates/certificate.yaml#L8

Please fix that before releasing